LICENSE.md

```diff
@@ -0,0 +1,21 @@
+# MIT License
+
+**Copyright (c) 2021 justgoodin**
```

Comment:
what's is justgoodin? and why are we copyrighting this to it?

Reply:
Generated it using  GitHub's license generator to make it fully permissive. Somehow they added this, I didn't notice.